File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Dãy số đẹp 1/DAYDEP1.cs
using System;

namespace SODEP1
{
    publoc class Program
    {
        static void Main(string[] args)
        {
            int s = 0;
            int x;

            while (int.TryParse(Console.ReadLine(), out x))
            {
                s += x;
                if (x == 0)
                    break;
            }

            if (s % 10 == 0)
            {
                Console.WriteLine("DEP");
            }
            else if (s % 10 == 5)
            {
                Console.WriteLine("TRUNGBINH");
            }
            else
            {
                Console.WriteLine("XAU");
            }
        }
    }
}

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Dãy số đẹp 1/DAYDEP1.cpp
#include<bits/stdc++.h>
using namespace std;
int main ()
{
    int s = 0;
    int x;
    while(cin >> x){
        s += x;
        if(x == 0)
        break;
    }
    if(s % 10 == 0){
        cout << "DEP";
        return 0;
    }
    if(s % 10 == 5){
        cout << "TRUNGBINH";
        return 0;
    }
    cout << "XAU";
    return 0;
}